web/src/engine/websites/templates/FlatManga.ts

```diff
@@ -0,0 +1,110 @@
+﻿import { AddAntiScrapingDetection, FetchRedirection } from "../../platform/AntiScrapingDetection";
+import { FetchCSS } from "../../platform/FetchProvider";
+import { DecoratableMangaScraper, type Chapter, type MangaScraper } from "../../providers/MangaPlugin";
+import { Page } from "../../providers/MangaPlugin";
+import * as Common from '../decorators/Common';
+
+AddAntiScrapingDetection(async (invoke) => {
+    const result = await invoke<boolean>(`document.documentElement.innerHTML.includes('ct_anti_ddos_key')`);// Sample => Mangagun, NicoManga, Rawinu, Weloma, WeloveManga
+    if (result) {
+        await new Promise(resolve => setTimeout(resolve, 3000));
```

Comment:
I don't understand the delay at this point.
I think this is not necessary and may lead to errors in case the website navigates within these 3 seconds an the automatic redirection may not be performed afterwards.

Reply:
Anti Ddoss is delaying the redirection after check is done.